packages/neuron-ui/src/widgets/RadioGroup/index.tsx

```diff
@@ -13,9 +13,18 @@ export interface RadioGroupProps {
   onChange?: (arg: string | number) => void
   defaultValue?: string | number
   itemClassName?: string
+  isHoverStyle?: boolean
+  className?: string
 }
 
-const RadioGroup = ({ defaultValue, options, onChange, itemClassName = '' }: RadioGroupProps) => {
+const RadioGroup = ({
+  defaultValue,
+  options,
+  onChange,
+  itemClassName = '',
+  isHoverStyle = true,
```

Comment:
The variable named `isHoverStyle` is confusing.
IMO, when hovering on the element, it will be hovering. Now it can be controlled by the component's property?